### ISSUE TITLE
Refer to form in Queue Manager

### ISSUE THREAD

**Issue by bartrylant** [verbb/formie#1103]
### What are you trying to do?

When a Freeform task fails (e.g. when there is an issue with MailChimp integration), the error message in the Queue Manager refers to the uid of the form causing the error. However, you can't view uid's in the control panel so you don't know which form causes the error.

### What's your proposed solution?

Refer to the form name / form id / form handle in the error message.

**Comment by engram-design** [comment]
Updated the descriptions of queue jobs to include the form title/name. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.13](https://github.com/verbb/formie/releases/tag/2.0.13)
